train.py

```diff
@@ -40,54 +44,51 @@ def main(args):
         train_loss = None
         validation_acc = 0
         validation_loss = None
-        split_val = int(len(dataloader) * 0.8)
-        for batch_num, batch in enumerate(tqdm(dataloader)):
-            if batch_num < split_val:
-                optimizer.zero_grad()
-                
-                img, labels = batch
+        for train_idx, train_batch in enumerate(tqdm(train_dataloader)):
+            optimizer.zero_grad()
+            
+            img, labels = train_batch
+            labels = list(labels)
+            
+            logits, loss = classifier(img.to(device), labels, device)
+            
+            for idx in range(len(labels)):
+                if labels[idx] == logits[idx]:
+                    train_acc = train_acc + 1
+            
+            if train_loss is None:
+                train_loss = loss
+            else:
+                train_loss = torch.mean(torch.stack([train_loss, loss]))
+            
+            if train_idx%10 == 0:
+                print(f"\nTraining accuracy : {train_acc}/{(train_idx + 1) * batch_size},\tTrining Loss : {train_loss}")
```

Comment:
```suggestion
                print(f"\n{train_idx}/{len(train_dataloader)}th iteration : Training accuracy : {(train_acc/((train_idx + 1) * batch_size))*100:.2f}%,\tTraining Loss : {train_loss}")
```

This print-out line will be fancier.